ui/src/job/JobListTable.js

```diff
@@ -94,10 +94,6 @@ const JobListTable = ({ projectId, modelId, jobs, isLoaded, error, fetchJobs })
     return ["pending", "running"].includes(status)
   }
 
-  const isTerminatedJob = function(status){
-    return ["terminating", "terminated"].includes(status)
```

Comment:
Should we still hide the Terminate / Delete buttons for "Terminating" state?

Reply:
Hmm, but that behaviour's going to be different from Turing then because in Turing be simply disable the button when in the `terminating` state

Reply:
Didn't realise that. Makes sense. And on second thoughts, "disabled" is better than hiding, for this scenario.